mlflow/transformers/__init__.py

```diff
@@ -1319,7 +1319,7 @@ def _should_add_pyfunc_to_model(pipeline) -> bool:
         "ZeroShotAudioClassificationPipeline",
     ]
 
-    impermissible_attrs = {"image_processor"}
+    impermissible_attrs = {""}
```

Comment:
1322 to 1326 can be deleted as it's a no-op.

Reply:
deleted

Reply:
made chages as per requested